### ISSUE TITLE
core acceptance tests failing in CI

### ISSUE THREAD

**Issue by phil-davis** [owncloud/encryption#302]
https://drone.owncloud.com/owncloud/encryption/2045/42/16
`apiShareManagementToRoot/acceptShares.feature:727`

https://drone.owncloud.com/owncloud/encryption/2045/43/16
`apiShareManagementToShares/acceptShares.feature:494`

https://drone.owncloud.com/owncloud/encryption/2045/64/16
```
apiTrashbinRestore/trashbinRestore.feature:161
apiTrashbinRestore/trashbinRestore.feature:177
```

https://drone.owncloud.com/owncloud/encryption/2045/65/16
```
unsh: Total unexpected failed scenarios throughout the test run:
apiVersions/fileVersions.feature:70
apiVersions/fileVersions.feature:78
apiVersions/fileVersions.feature:97
apiVersions/fileVersions.feature:101
apiVersions/fileVersions.feature:105
apiVersions/fileVersions.feature:116
apiVersions/fileVersions.feature:176
apiVersions/fileVersions.feature:188
apiVersions/fileVersions.feature:200
apiVersions/fileVersions.feature:212
apiVersions/fileVersions.feature:224
apiVersions/fileVersions.feature:236
apiVersions/fileVersions.feature:248
apiVersions/fileVersions.feature:262
apiVersions/fileVersions.feature:275
apiVersions/fileVersions.feature:289
apiVersions/fileVersions.feature:400
apiVersions/fileVersions.feature:437
apiVersions/fileVersions.feature:455
apiVersions/fileVersionsSharingToShares.feature:55
apiVersions/fileVersionsSharingToShares.feature:68
apiVersions/fileVersionsSharingToShares.feature:81
apiVersions/fileVersionsSharingToShares.feature:94
apiVersions/fileVersionsSharingToShares.feature:107
apiVersions/fileVersionsSharingToShares.feature:120
apiVersions/fileVersionsSharingToShares.feature:133
apiVersions/fileVersionsSharingToShares.feature:148
apiVersions/fileVersionsSharingToShares.feature:178
apiVersions/fileVersionsSharingToSharesIssue36228.feature:10
apiVersions/fileVersionsSharingToSharesIssue38027.feature:14
runsh: There were no unexpected success.
```

https://drone.owncloud.com/owncloud/encryption/2045/134/11
```
runsh: Total unexpected failed scenarios throughout the test run:
cliLocalStorage/createLocalStorageForGroups.feature:58
cliLocalStorage/createLocalStorageForUsers.feature:81
runsh: There were no unexpected success.
```

https://drone.owncloud.com/owncloud/encryption/2045/141/17
```
webUIFiles/versions.feature:25
```

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/encryption/2045/42/16
```
  Scenario: shares exist after restoring already shared file to a previous version                # /var/www/owncloud/testrunner/tests/acceptance/features/apiShareManagementToRoot/acceptShares.feature:727
    Given user "Alice" has uploaded file with content "Test Content." to "/toShareFile.txt"       # FeatureContext::userHasUploadedAFileWithContentTo()
    And user "Alice" has uploaded file with content "Content Test Updated." to "/toShareFile.txt" # FeatureContext::userHasUploadedAFileWithContentTo()
    And user "Alice" has shared file "/toShareFile.txt" with user "Brian"                         # FeatureContext::userHasSharedFileWithUserUsingTheSharingApi()
    When user "Alice" restores version index "1" of file "/toShareFile.txt" using the WebDAV API  # FilesVersionsContext::userRestoresVersionIndexOfFile()
    Then the content of file "/toShareFile.txt" for user "Alice" should be "Test Content."        # FeatureContext::contentOfFileForUserShouldBe()
      The downloaded content was expected to be 'Test Content.', but actually is '<?xml version'. HTTP status was 500
      Failed asserting that two strings are equal.
      --- Expected
      +++ Actual
      @@ @@
      -'Test Content.'
      +'<?xml version'
    And the content of file "/toShareFile.txt" for user "Brian" should be "Test Content."         # FeatureContext::contentOfFileForUserShouldBe()
```

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/encryption/2045/43/16
```
  Scenario: shares exist after restoring already shared file to a previous version                # /var/www/owncloud/testrunner/tests/acceptance/features/apiShareManagementToShares/acceptShares.feature:494
    And user "Alice" has uploaded file with content "Test Content." to "/toShareFile.txt"         # FeatureContext::userHasUploadedAFileWithContentTo()
    And user "Alice" has uploaded file with content "Content Test Updated." to "/toShareFile.txt" # FeatureContext::userHasUploadedAFileWithContentTo()
    And user "Alice" has shared file "/toShareFile.txt" with user "Brian"                         # FeatureContext::userHasSharedFileWithUserUsingTheSharingApi()
    And user "Brian" has accepted share "/toShareFile.txt" offered by user "Alice"                # FeatureContext::userHasReactedToShareOfferedBy()
    When user "Alice" restores version index "1" of file "/toShareFile.txt" using the WebDAV API  # FilesVersionsContext::userRestoresVersionIndexOfFile()
    Then the content of file "/toShareFile.txt" for user "Alice" should be "Test Content."        # FeatureContext::contentOfFileForUserShouldBe()
      The downloaded content was expected to be 'Test Content.', but actually is '<?xml version'. HTTP status was 500
      Failed asserting that two strings are equal.
      --- Expected
      +++ Actual
      @@ @@
      -'Test Content.'
      +'<?xml version'
    And the content of file "/Shares/toShareFile.txt" for user "Brian" should be "Test Content."  # FeatureContext::contentOfFileForUserShouldBe()
```

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/encryption/2045/64/16
```
  Scenario: Deleting an updated file into external storage moves it to the trashbin and can be restored                                        # /var/www/owncloud/testrunner/tests/acceptance/features/apiTrashbinRestore/trashbinRestore.feature:161
    Given using old DAV path                                                                                                                   # FeatureContext::usingOldOrNewDavPath()
    And the administrator has invoked occ command "files:scan --all"                                                                           # OccContext::theAdministratorHasInvokedOccCommand()
    And user "Alice" has created folder "/local_storage/tmp"                                                                                   # FeatureContext::userHasCreatedFolder()
    And user "Alice" has moved file "/textfile0.txt" to "/local_storage/tmp/textfile0.txt"                                                     # FeatureContext::userHasMovedFile()
    And user "Alice" has uploaded chunk file "1" of "1" with "AA" to "/local_storage/tmp/textfile0.txt"                                        # FeatureContext::userHasUploadedChunkedFile()
    And user "Alice" has deleted file "/local_storage/tmp/textfile0.txt"                                                                       # FeatureContext::userHasDeletedFile()
      HTTP status code was not 204 while trying to delete file '/local_storage/tmp/textfile0.txt' for user 'Alice'
      Failed asserting that an array contains 500.
```

```


Scenario: Deleting an updated file into external storage moves it to the trashbin and can be restored with new chunking                      # /var/www/owncloud/testrunner/tests/acceptance/features/apiTrashbinRestore/trashbinRestore.feature:177 | 475s
-- | --
432 | Given using new DAV path                                                                                                                   # FeatureContext::usingOldOrNewDavPath() | 480s
433 | And the administrator has invoked occ command "files:scan --all"                                                                           # OccContext::theAdministratorHasInvokedOccCommand() | 481s
434 | And user "Alice" has created folder "/local_storage/tmp"                                                                                   # FeatureContext::userHasCreatedFolder() | 482s
435 | And user "Alice" has moved file "/textfile0.txt" to "/local_storage/tmp/textfile0.txt"                                                     # FeatureContext::userHasMovedFile() | 482s
436 | And user "Alice" has uploaded the following chunks to "/local_storage/tmp/textfile0.txt" with new chunking                                 # FeatureContext::userHasUploadedTheFollowingChunksUsingNewChunking() | 483s
437 | \| number \| content \| | 483s
438 | \| 1      \| AA      \| | 483s
439 | And user "Alice" has deleted file "/local_storage/tmp/textfile0.txt"                                                                       # FeatureContext::userHasDeletedFile() | 485s
440 | HTTP status code was not 204 while trying to delete file '/local_storage/tmp/textfile0.txt' for user 'Alice' | 485s
441 | Failed asserting that an array contains 500.
```

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/encryption/2045/65/16
```
  Scenario: Restore a file and check, if the content is now in the current file               # /var/www/owncloud/testrunner/tests/acceptance/features/apiVersions/fileVersions.feature:70
    Given user "Alice" has uploaded file with content "Test Content." to "/davtest.txt"       # FeatureContext::userHasUploadedAFileWithContentTo()
    And user "Alice" has uploaded file with content "Content Test Updated." to "/davtest.txt" # FeatureContext::userHasUploadedAFileWithContentTo()
    And the version folder of file "/davtest.txt" for user "Alice" should contain "1" element # FilesVersionsContext::theVersionFolderOfFileShouldContainElements()
    When user "Alice" restores version index "1" of file "/davtest.txt" using the WebDAV API  # FilesVersionsContext::userRestoresVersionIndexOfFile()
    Then the content of file "/davtest.txt" for user "Alice" should be "Test Content."        # FeatureContext::contentOfFileForUserShouldBe()
      The downloaded content was expected to be 'Test Content.', but actually is '<?xml version'. HTTP status was 500
      Failed asserting that two strings are equal.
      --- Expected
      +++ Actual
      @@ @@
      -'Test Content.'
      +'<?xml version'
```

And more similar test fails.

**Comment by phil-davis** [comment]
Fixed - PR #300 has been merged.
